### PR TITLE
First Icon

### PR DESCRIPTION
**Pull request by Fatih20**
#1 
It represents a computer. The bottom part is supposed to be the grill but it's also a play on statistic. Feel free to give any feedback.

### CODE REVIEW

File: data/icons/128/con.github.PlugaruT.wingpanel-monitor.svg [hunk 1]
```diff
@@ -0,0 +1,1335 @@
...
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="128"
+   height="128"
+   viewBox="0 0 33.866666 33.866668"
+   version="1.1"
+   id="svg8"
+   inkscape:version="0.92.3 (2405546, 2018-03-11)"
+   sodipodi:docname="128.svg"
+   inkscape:export-filename="/home/fatih/Documents/Github Stuff/wingpanel-monitor/data/icons/128x128.png"
```
Comment (PlugaruT) [inline data/icons/128/con.github.PlugaruT.wingpanel-monitor.svg]: Can we get rid of this please?

File: data/icons/128/con.github.PlugaruT.wingpanel-monitor.svg [hunk 2]
```diff
@@ -0,0 +1,1335 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (PlugaruT) [inline data/icons/128/con.github.PlugaruT.wingpanel-monitor.svg]: Also, there are some typos in the file names, `con.github.PlugaruT.wingpanel-monitor.svg` should be `com.github.plugarut.wingpanel-monitor.svg`

Comment (Fatih20) [reply data/icons/128/con.github.PlugaruT.wingpanel-monitor.svg]: Sure.

File: data/icons/128/con.github.PlugaruT.wingpanel-monitor.svg (hunk 1 above)
Comment (Fatih20) [reply data/icons/128/con.github.PlugaruT.wingpanel-monitor.svg]: What? You're not supposed to read the code lol. Look at the svg, not read them.

Comment (PlugaruT) [reply data/icons/128/con.github.PlugaruT.wingpanel-monitor.svg]: Yes, I know and I've looked at the icon but I also noticed this :man_shrugging:

Comment (Fatih20) [reply data/icons/128/con.github.PlugaruT.wingpanel-monitor.svg]: Right.

**Comment by PlugaruT** [comment]
Also, I've tried to stick in into the app, and the icon in the dock looks wrong. It seems that it has a background, when it's not supposed to have one.
image

**Comment by Fatih20** [comment]
Seems like there's some technical mistake I did. I'll get it fixed soon. Find other flaws with the icon so I can bundle them up.

**Comment by PlugaruT** [comment]
> Seems like there's some technical mistake I did. I'll get it fixed soon. Find other flaws with the icon so I can bundle them up.

That's all I could find for now.

**Comment by PlugaruT** [comment]
@Fatih20, the icon looks good, but can I ask you one small thing, can you please make the height of the icon be the same as other eOS icons? 
image
It's higher than other apps and it doesn't look good in the dock.

**Comment by Fatih20** [comment]
I can't fit the icon to match the sizes of other eOS app icons because their heights also varies as shown by this screenshot below.
image

**Comment by PlugaruT** [comment]
Can you please make it have the same height as the calculator app? Right now, this one has the bigger height from any app I have installed. Thank you :bowing_man:

**Comment by PlugaruT** [comment]
Perfect :tada: Thank you for your help @Fatih20

**Comment by Fatih20** [comment]
Thanks! Please credit me in the patch note and consider buying me a coffee on liberapay!

**Pull request merged**